app/client/src/components/pages/Community.Tabs.Overview.js

```diff
@@ -612,6 +589,18 @@ function MonitoringAndSensorsTab({
     ],
   );
 
+  if (
+    streamgagesStatus === 'failure' &&
+    monitoringLocationsStatus === 'failure'
+  ) {
+    return (
+      <div css={errorBoxStyles}>
+        <p>{streamgagesError}</p>
+        <p>{monitoringError}</p>
+      </div>
+    );
+  }
```

Comment:
```suggestion
  if (
    streamgagesStatus === 'failure' ||
    monitoringLocationsStatus === 'failure'
  ) {
    return (
      <div css={errorBoxStyles}>
        {streamgagesStatus === 'failure'  && <p>{streamgagesError}</p>}
        {monitoringLocationsStatus === 'failure' && <p>{monitoringError}</p>}
      </div>
    );
  }
```

I think this needs to be tweaked a little bit so the error message is displayed if just one of these services is down. The above should do it (although I didn't test it).

Reply:
It looks like an error will still show below, starting in line 657, if there is only one failure, but this block just returns when both are failed so that the "success" content isn't rendered.

Reply:
Ok. The one below isn't working for situations where there are 0 monitoring locations. So I think the second set of messages below need to be pulled out of the `{allMonitoringAndSensors.length > 0` block.

![image](https://user-images.githubusercontent.com/46329268/228555213-326021b0-08e5-4de8-951a-c72b66eba606.png)

Reply:
That's a good point, it should be remedied now.

Reply:
Thanks for making that change. Everything looks good now.